pkg/interop/account/account.go

```diff
@@ -1,23 +1,36 @@
+/*
+Package account provides getter functions for Account interop structure.
+To use these functions you need to get an Account first via blockchain.GetAccount
+call.
+*/
 package account
 
-// Package account provides function signatures that can be used inside
-// smart contracts that are written in the neo-go framework.
-
-// Account stubs a NEO account type.
+// Account represents NEO account type that is used in interop functions, it's
+// an opaque data structure that you can get data from only using functions from
+// this package. It's similar in function to the Account class in the Neo .net
+// framework.
 type Account struct{}
 
-// GetScriptHash returns the script hash of the given account.
+// GetScriptHash returns the script hash of the given Account (20 bytes in BE
+// representation). It uses `Neo.Account.GetBalance` syscall internally.
 func GetScriptHash(a Account) []byte {
 	return nil
 }
 
-// GetVotes returns the votes of the given account which should be a slice of
-// public key raw bytes.
+// GetVotes returns current votes of the given account represented as a slice of
+// public keys. Keys are serialized into byte slices in their compressed form (33
+// bytes long each). This function uses `Neo.Account.GetVotes` syscall
+// internally.
 func GetVotes(a Account) [][]byte {
```

Comment:
I was thinking about making this interops a method of an `Account` struct. Not here, of course.

Reply:
#296. And #941. But it's not a priority for now.

Reply:
And not for 2.x also, I think.